sabre1041/managing-ocp-install-beyond
# doing a restructure of the docs

This is a major restructure of the documentation.  
* created a backup directory in labs/ to move the cake-php
* added the prometheus / grafana lab
* moved cloudforms to the end

To test this
* Review the README links and make sure they all work
* review all headers and footers in each doc to make sure they reference the correct location
* Click all the links at the bottom of each doc and make sure they take you to the correct chapter.
* make sure the overall flow is what we want.
